opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/aggregation/AggregationQueryBuilderTest.java

```diff
@@ -71,24 +73,26 @@ void set_up() {
   void should_build_composite_aggregation_for_field_reference() {
     assertEquals(format(
         "{%n"
-            + "  \"composite_buckets\" : {%n"
-            + "    \"composite\" : {%n"
-            + "      \"size\" : 1000,%n"
-            + "      \"sources\" : [ {%n"
-            + "        \"name\" : {%n"
-            + "          \"terms\" : {%n"
-            + "            \"field\" : \"name\",%n"
-            + "            \"missing_bucket\" : true,%n"
-            + "            \"missing_order\" : \"first\",%n"
-            + "            \"order\" : \"asc\"%n"
+            + "  \"composite_buckets\": {%n"
+            + "    \"composite\": {%n"
+            + "      \"size\": 1000,%n"
+            + "      \"sources\": [%n"
+            + "        {%n"
```

Comment:
What was actually changed here?

Reply:
Minor spacing and formatting changes that stem from a difference between GSON and jackson's pretty printing methods